### PR TITLE
PD-248: Update clusterdomain status version

### PR DESCRIPTION
**Pull request by andreipope**
Signed-off-by: Andrei Popescu <andrei.popescu@gmail.com>

### CODE REVIEW

**Comment by nathan-px** [comment]
@andrei I don't like that this topic is just dumped into the prerequisites out of context and with no title. Sometimes it's even just thrown in between two bullet points in a prerequisite list:

image

We have two options here, we can make this a standalone topic that we link to in the prerequisites list, or we can properly title it and move it out of the prerequisites into the first step. Since these steps don't seem to be limited to just DR, I'm partial to making them a standalone topic.

**Comment by andreipope** [comment]
> see earlier comment

@nathan-px I see your point. We could make a page for Stork, but it probably needs a bit of planning and more content. For now, I'd just move this into the first step. Thoughts?

File: content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md [hunk 1]
```diff
@@ -7,18 +7,28 @@ The following steps can be used to download `storkctl`:
...
 
     ```text
-    curl http://openstorage-stork.s3-website-us-east-1.amazonaws.com/storkctl/latest/darwin/storkctl -o storkctl &&
+    STORK_POD=`kubectl get pods -n kube-system -l name=stork -o json | jq .items[0].metadata.name -r` &&
+    kubectl cp -n kube-system $STORK_POD:/storkctl/darwin/storkctl . &&
     sudo mv storkctl /usr/local/bin &&
     sudo chmod +x /usr/local/bin/storkctl
     ```
 
+<!--@SanjayN: Is this going to work on Windows? I don't have an environment to test it-->
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md]: @sanjaynaikwadi Please see this comment.

Comment (andreipope) [inline content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md]: @sanjaynaikwadi Please see this comment.

**Comment by andreipope** [comment]
> move it out of the prerequisites into the first step.

@nathan-px It is now moved into the first step. Please see my earlier comment about creating a standalone page.

**Comment by nathan-px** [comment]
> > see earlier comment
> 
> @nathan-px I see your point. We could make a page for Stork, but it probably needs a bit of planning and more content. For now, I'd just move this into the first step. Thoughts?

Yeah, that's an acceptable compromise.

File: content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md]: @sanjaynaikwadi Kind reminder.

Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md]: @sanjaynaikwadi

Comment (sanjaynaikwadi) [reply content/portworx-install-with-kubernetes/disaster-recovery/shared/stork-helper.md]: @andreipope - I don't have any windows machines nor I have many folks using on windows.
We are good with Linux & Mac part.

**Comment by andreipope** [comment]
@sanjaynaikwadi @disrani-px @nathan-px I just removed the instructions for Windows.
Please let me know if you require any other changes.
Thanks!

Screenshot 2020-01-17 at 09 30 02

**Comment by andreipope** [comment]
> New changes are good to go. I have verified and those are working.

Thanks @sanjaynaikwadi !

**Comment by andreipope** [comment]
> Please don't remove the changes for windows

@disrani-px I brought back the changes for Windows. Please let me know if you require any other changes.

File: content/shared/portworx-install-with-kubernetes-disaster-recovery-stork-helper.md [hunk 2]
```diff
@@ -7,25 +7,28 @@ The following steps can be used to download `storkctl` from the Stork pod:
...
     sudo mv storkctl /usr/local/bin &&
     sudo chmod +x /usr/local/bin/storkctl
     ```
+
 * OS X:
 
     ```text
-    STORK_POD=$(kubectl get pods -n kube-system -l name=stork -o jsonpath='{.items[0].metadata.name}') &&
-    kubectl cp -n kube-system $STORK_POD:/storkctl/darwin/storkctl ./storkctl
+    STORK_POD=`kubectl get pods -n kube-system -l name=stork -o json | jq .items[0].metadata.name -r` &&
```
Comment (disrani-px) [inline content/shared/portworx-install-with-kubernetes-disaster-recovery-stork-helper.md]: Not everyone has `jq` installed, so the previous version was better

File: content/shared/portworx-install-with-kubernetes-disaster-recovery-stork-helper.md [hunk 3]
```diff
@@ -7,25 +7,28 @@ The following steps can be used to download `storkctl` from the Stork pod:
...
 * Windows:
 
-    * Copy `storkctl.exe` from the stork pod:
-    ```text
-    STORK_POD=$(kubectl get pods -n kube-system -l name=stork -o jsonpath='{.items[0].metadata.name}') &&
-    kubectl cp -n kube-system $STORK_POD:/storkctl/windows/storkctl.exe ./storkctl.exe
-    ```
-    * Move `storkctl.exe` to a directory in your PATH
+  * Copy `storkctl.exe` from the Stork pod:
+  ```text
```
Comment (nathan-px) [inline content/shared/portworx-install-with-kubernetes-disaster-recovery-stork-helper.md]:
```suggestion

  ```text
```

**Comment by nathan-px** [comment]
#685

**Pull request closed without merging**